articles/configure-your-card-previews.mdx

```diff
@@ -159,6 +161,27 @@ preview:
 ```
 </comp.CodeBlock>
 
+### Templating in previews
+
+When using the `template` key in a preview's cascading options, you can reference keys from the object. If **any** placeholder resolves to an empty result, the whole preview entry is skipped, and the next preview entry is run.
+
+See [this article for more information about using template strings](/documentation/articles/configuring-with-placeholders/), as well as a list of available filters.
+
+Take the following config for example. The preview will show 1 line of metadata with the "person" icon. The text for this metadata will show "By Ella" if the object has an object `author` with a `name` field set to "Ella". If `author.name` isn't set, the text will show "No author".
+
+<comp.CodeBlock language="YAML" source="cloudcannon.config.yml">
```

Comment:
`preview` isn't a top-level key in `cloudcannon.config.yml`, so I'd say we either want to relabel it, or nest the example inside a `collection_config` entry.